### PR TITLE
Update the README

### LINKED ISSUE #8: unable to get this working on El Capitan ( mac osX )
Installed from scratch, got linter-rubocop working.  Compared config with colleague on Yosimite, and the config looks all good.  Using "known bad" code for foodcritic, but foodcritic just does not run.  Rubucop runs every time, and even if I fix the rubucop things, foodcritic still does not run.   I installed chefdk, so running from the command line foodcritic works just fine on the file ( finds x4 issues).   

Also blew away the ~/.atom and started again.  Still no foodcritic.  Am pretty sure it is related to El Capitan.   Appreciate if anyone has seen same or has advice on what to try next.

file is below:-

```
file '/tmp/something' do
  punter 'root'
  action :create
end
```

### PR DESCRIPTION
**Pull request by Arcanemagus**
Clean up the text a bit, update links, remove the Settings section, and split out the CONTRIBUTING into a file of its own.

Fixes #8.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,31 @@
 linter-foodcritic
 =========================
 
-This linter plugin for [Linter](https://github.com/AtomLinter/Linter) provides an interface to [foodcritic](https://github.com/AtomLinter/linter-foodcritic). It will be used with files that have the “Ruby” syntax.
+This linter plugin for [Linter](https://github.com/atom-community/linter)
```
Comment (steelbrain) [inline README.md:4]: It would've been nice if it was `This is a [linter][linter] provider for [foodcritic][foodcritic]`

Comment (Arcanemagus) [inline README.md:4]: Do you mean `Linter` should have been `linter`?

Comment (steelbrain) [inline README.md:4]: I mean instead of using word `linter` twice it should've been used once IMHO. but I don't mind the way it is right now

Comment (Arcanemagus) [inline README.md:4]: I think I see what you mean, I was planning on further cleaning up the actual text when (if...) further work was done on this like auto-installation of `linter`.

**Pull request merged**